app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/util/NavUtils.kt

```diff
@@ -0,0 +1,21 @@
+package com.bottlerocketstudios.brarchitecture.ui.util
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.navigation.NavController
+import androidx.navigation.NavOptionsBuilder
+import com.bottlerocketstudios.brarchitecture.ui.Routes
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.FlowCollector
+
+fun NavOptionsBuilder.popToMainInclusive() {
```

Comment:
This function could live in MainNavGraph.... @br-jared-stockton What do you think?